bedrock/base/templates/macros.html

```diff
@@ -145,9 +145,10 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
 {%- endmacro %}
 
 {# Docs: https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#linking-to-accountsfirefoxcom #}
-{% macro fxa_link_fragment(entrypoint, service_type='sync', action='signin', utm_campaign=None, utm_content=None, utm_term=None) -%}
+{% macro fxa_link_fragment(entrypoint, entrypoint_experiment, entrypoint_variation, service_type='sync', action='signin', form_type='button', utm_campaign=None, utm_content=None, utm_term=None) -%}
```

Comment:
Looks like there are still a few more places where you could set a default value for the new parameters